data/campaigns/Winds_of_Fate/scenarios/04_Islands.cfg

```diff
@@ -0,0 +1,609 @@
+            facing=sw
+        [/leader]
+    [/side]
+
+    [side]
+        side=3
+        controller=ai
+        defeat_condition=no_units_left
+        recruit=Naga Dirkfang, Naga Fighter
+        gold=22
+        {INCOME 22 22 22}
+        team_name=nagas
+        user_team_name= _ "Serpents"
+        [leader]
+            id=Shussek
+            name= _ "Shussek"
+            type=Naga Warrior
+            facing=sw
+        [/leader]
+        [ai]
+            aggression=0.6
+            passive_leader=yes # so he doesn't get himself killed
+        [/ai]
+    [/side]
+
+    [side]
+        side=4
+        controller=ai
+        defeat_condition=no_units_left
+        recruit=Merman Fighter, Merman Hunter, Mermaid Initiate
+        gold=32
+        {INCOME 16 32 48}
+        team_name=merfolk
+        user_team_name= _ "Merfolk"
+        [leader]
+            id=Poseira
+            name= _ "Poseira"
+            type=Merman Spearman
+            facing=se
+        [/leader]
+        [ai]
+            aggression=0.6
+            passive_leader=yes # so he doesn't get himself killed
+        [/ai]
+    [/side]
+
+    # wmllint: validate-on
+
+    [event]
+        name=prestart
+        {MODIFY_UNIT side=1 facing se}
+        [hide_unit]
+            side=1
+        [/hide_unit]
+        # Clashers will be returned in a future scenario when the drakes can forge weapons & armor
+        [store_unit]
+            [filter]
+                side=1
+                type_adv_tree=Drake Clasher
+            [/filter]
+            variable=clashers
+            kill=yes
+        [/store_unit]
+        [disallow_recruit]
+            side=1
+            type=Drake Clasher
+        [/disallow_recruit]
+        {VARIABLE drakes_killed_nagas no}
+        [objectives]
+            side=1
+            [objective]
+                description= _ "Be holding all villages at the third day's end. (Turn 18)"
+                condition=win
+            [/objective]
+            [objective]
+                description= _ "Fail to hold all villages at the third day's end. (Turn 18)"
+                condition=lose
+            [/objective]
+            [objective]
+                description= _ "Death of Gorlack"
+                condition=lose
+            [/objective]
+            [objective]
+                description= _ "Death of Reshan"
+                condition=lose
+            [/objective]
+            [gold_carryover]
+                carryover_percentage=0
+            [/gold_carryover]
+            [note]
+                description= _ "If we kill none of the serpentine forces, we can convince them to ally with us by moving Reshan adjacent to their leader."
+            [/note]
+            [note]
+                description= _ "The clasher caste has no role to play in this offshore scenario. They will not be participating."
+            [/note]
+            [note]
+                description= _ "A summer time schedule is in effect."
```

Comment:
It wasn't clear to me what the summer time schedule meant until I looked it up in the Help. I suppose that's all that's needed to indicate the change from normal time of day mechanics. I think there are other scenarios later on that also differ from the normal schedule but don't have a similar note, though.

Reply:
I checked them over to be sure; it looks like S4 and S9 are the only scenarios using unusual time schedules at this point, and each has a note in the objectives about this.

I could extend the note to say: "A summer time schedule is in effect. Daytime is longer."

Reply:
Okay, I must have missed the late note. Thanks for clarifying.

Reply:
Extended the note to:
"A summer time schedule is in effect. Daytime is longer."

And for S9:
"A winter time schedule is in effect. Nighttime is longer."